source/OptimaJet.DWKit.StarterApplication/Services/BuildEngine/BuildEngineBuildService.cs

```diff
@@ -272,7 +272,9 @@ protected async Task BuildCreationFailedAsync(Product product, BuildResponse bui
         {
             ClearRecurringJob(product.Id);
             await SaveArtifacts(product, buildEngineBuild, false);
-            var buildEngineUrl = product.Project.Organization.BuildEngineUrl + "/build-admin/view?id=" + product.WorkflowBuildId.ToString();
+            var buildEngineEndpoint = GetBuildEngineEndpoint(product.Project.Organization);
```

Comment:
I assume that `GetBuildEngineEndpoint` get the default BuildEngine if the Organization is using the default.

Reply:
Correct